features/sync/current_branch/feature_branch/override/sync_strategy_override.feature

```diff
@@ -0,0 +1,54 @@
+Feature: override the configured sync-feature-strategy
+
+  Background:
+    Given a Git repo with origin
+    And the branches
+      | NAME    | TYPE    | PARENT | LOCATIONS     |
+      | feature | feature | main   | local, origin |
+    And Git Town setting "sync-feature-strategy" is "merge"
```

Comment:
I just shipped a change that will break this. Here is the fix:

```suggestion
    And Git setting "git-town.sync-feature-strategy" is "merge"
```

Reply:
Thanks!